.gitignore

```diff
@@ -140,4 +140,8 @@ dist
 .yarn/unplugged
 .yarn/build-state.yml
 .yarn/install-state.gz
-.pnp.*
\ No newline at end of file
+.pnp.*
```

Comment:
tbh this should go to your global git ignore.
Here is a good blog post about it https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/